### PR TITLE
[n/a] add Tabs documentation

### PR DESCRIPTION
**Pull request by gregkohn**
🚀  https://code.viget.com/stimulus-controllers/tabs/

### CODE REVIEW

**Review by davist11 — APPROVED** [review]

File: src/pages/tabs.astro [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+      <div slot="code">
+        <Markdown>
+          ```jsx
+          <div data-controller="tabs">
+            <ul
+              class="flex"
+              data-action="keyup->tabs#cycleTabs"
+              role="tablist"
+            >
+              {['Tab 1', 'Tab 2', 'Tab 3'].map((text, i) => (
```
Comment (joshuapease) [inline src/pages/tabs.astro:141]: Super minor, but the JSX here might potentially trip up newer folks who aren't familiar what parts are Stimulus syntax and what's JSX.

**Review by joshuapease — APPROVED** [review]

**Review by jeremyfrank — APPROVED** [review]

File: src/pages/tabs.astro (hunk 1 above)
Comment (gregkohn) [reply src/pages/tabs.astro:141]: fair, i struggled with that. should we commit to twig demo code? the hold up for me was i couldnt actually (or easily) verify the twig code the way i could the jsx code

Comment (gregkohn) [reply src/pages/tabs.astro:141]: oh and then i thought i could just manually do multiple elements but i figured the loop logic was helpful for the attributes even if it was in jsx 🤷

**Review by olets — COMMENTED** [review]
This is great. Thoughts about improving accessibility?

- arrow, home, and end keys to switch between tabs
- tabindex="0" on the active tabpanel
- aria-controls / aria-labelledby between tabs and tabpanels
- aria-selected on tabs
- aria-label on the tablist

Refs:
- https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html
- https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html

File: src/pages/tabs.astro (hunk 1 above)
Comment (olets) [reply src/pages/tabs.astro:141]: somewhat related, entirely nit/subjective: did you consider having everything in a single array? like

```
tabsAndTabpanels = [
  {
    tabContent: …,
    tabpanelContent: …,
  },
  …
]
```

**Comment by gregkohn** [comment]
> * arrow, home, and end keys to switch between tabs

👍  you just mean home & end are missing?

> * tabindex="0" on the active tabpanel

👍 

> * aria-controls / aria-labelledby between tabs and tabpanels

`aria-labelledby` 👍 
@ten1seven was advising against `aria-controls` after was [dragged by Heydon](https://heydonworks.com/article/aria-controls-is-poop/). w3c has [an open ticket](https://github.com/w3c/aria/issues/995). i donno how we feel about it now but i haven't been using it.

> * aria-selected on tabs

this is already there unless i'm missing something?

> * aria-label on the tablist

👍

**Comment by olets** [comment]
> just home & end

huh. correct. don't know what I was looking at ha

> aria-controls

oh interesting I missed that

> aria-selected

"huh. correct. don't know what I was looking at ha" take 2 🤷‍♂️

**Pull request merged**